Makefile

```diff
@@ -124,3 +133,21 @@ dist: all
 	@ tar czf $(DISTDIR)/$(DISTNAME).tar.gz $(DISTNAME)
 	@ rm -rf $(DISTNAME)
 	@ echo "Distribution built; see 'tar tzf $(DISTDIR)/$(DISTNAME).tar.gz'"
+
+distsingle:
+	@ rm -rf $(DISTNAME) 2>/dev/null || true
```

Comment:
Move this to a separate `distclean` task and remove `|| true`, since it will be only called manually and it has to fail.

See https://www.gnu.org/prep/standards/html_node/Standard-Targets.html